module/damage/damagechat.js

```diff
@@ -91,10 +91,10 @@ export default class DamageChat {
 
   /**
    * Create the damage chat message.
-   * @param {GurpsActor|User} actor that rolled the damage.
+   * @param {Actor<"base" | ModuleSubtype>|User} actor that rolled the damage.
```

Comment:
I believe this should be Actor.Implementation instead

Reply:
I get TS compilation errors when I make that change. I'll leave it for now and we can resolve it in the next release.